person_class.rb

```diff
@@ -0,0 +1,51 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age? || @parent_permission ? true : false
```

Comment:
- [ ] We don't need to repeat ourselves here. The `@parent_permission` instance variable is either `true` or `false` so we don't need to check for it.